src/node/api.js

```diff
@@ -72,6 +75,33 @@ function createAPI (node, webhooks, interval=30) {
     }
 }
 
+function getAnonymousNeigbourNodeStats (nodeList) {
+    nodeList.peers.forEach(function(peer) {
+        if (peer.data.ip != null) {
+            peer.data.ip = getMD5Hash(peer.data.ip.toString);
+            peer.data.isAnonymous = true;
+        }
+    });
+    return nodeList.peers;
+}
+
+function getAnonymousNodeStats (node) {
+    var json = getNodeStats(node);
+    json.connectedPeers.forEach(function(node) {
+        if (node != null) {
+            if (node.ip != null) {
+                node.ip = getMD5Hash(node.ip.toString());
+                node.isAnonymous = true;
```

Comment:
node.isAnonymous here is enough. Since it's not the node itself, but a duplicate object, we can change it directly here.